### PR TITLE
View bookings

### PR DESCRIPTION
**Pull request by dereklance**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -9,6 +9,11 @@ const routes: Routes = [
     path: '',
     component: ToolbarComponent,
     children: [
+      {
+        path: '',
+        redirectTo: 'booking/flight',
+        pathMatch: 'full'
```
Comment (PatrickSmoothstack) [inline src/app/app-routing.module.ts:15]: Shouldn't you redirect the root path to the login page? Then on login, redirect to booking/flight?

File: src/app/manage-bookings/services/booking.service.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class BookingService {
+
+  private readonly apiUrl = environment.FLIGHTS_API;
+
+  constructor(private http: HttpClient) { }
+
+  private handleError(error: HttpErrorResponse): Observable<BookingResponse> {
```
Comment (PatrickSmoothstack) [inline src/app/manage-bookings/services/booking.service.ts]: Can you abstract this into a more reusable form? I see duplicated code in both your services in this commit.

**Review by aaronpxavier — APPROVED** [review]
Looks amazing.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (dereklance) [reply src/app/app-routing.module.ts:15]: Now handled by a route guard

File: src/app/manage-bookings/services/booking.service.ts (hunk 2 above)
Comment (dereklance) [reply src/app/manage-bookings/services/booking.service.ts]: Abstracted into generic type, reused across all services now

**Pull request merged**